src/index.js

```diff
@@ -45,3 +45,22 @@ exports.Queue = Plotly.Queue;
 
 // export d3 used in the bundle
 exports.d3 = require('d3');
+
+Plotly.register({
+    traces: {
+        bar: require('./traces/bar'),
```

Comment:
Can we go with `registerTraces` as the function name, and accept a single `traces` array (or single trace) as the argument? I don't like using `arguments` unless it's the only way because it isn't as easily portable (you would have to use Array.spread) and it doesn't add any clarity (in my opinion). It's great when you have multiple optional arguments after the main ones, but in this case I think it makes more sense to go with a standard array.